README.md

```diff
@@ -48,19 +48,42 @@ Giving the following output
 
 ## Example: Counting incoming Spanish traffic
 
+Create file /etc/geoip.nft
 ```
-  table filter {
-    include "./geoip-def-all.nft"
-    include "./geoip-ipv4.nft"
-    include "./geoip-ipv6.nft"
+  #!/usr/sbin/nft -f
+
+  table geoip {
+    include "geoip-def-all.nft" # fill absolute path
+    include "geoip-ipv4.nft" # fill absolute path
+    include "geoip-ipv6.nft" # fill absolute path
 
     chain input {
-                  type filter hook input priority filter; policy accept;
+                  type filter hook input priority 0; policy accept;
```

Comment:
I prefer if the example is not so distro-specific. What's the problem in Debian?

[nftables wiki](https://wiki.nftables.org/wiki-nftables/index.php/Configuring_chains#Base_chain_priority) states that `priority filter` has value 0.

Reply:
For example, I modify nftables.conf(change 0 to filter) and nft -f it. I will see:
```
/etc/nftables.conf:7:49-54: Error: syntax error, unexpected string, expecting - or number
                type filter hook input priority filter;
                                                              ^^^^^^
```
Before making this change, I have already check the wiki. I can't find anything (or I can't understand) about 'filter' mean 0.

But finally, I found 'Support for textual chain priorities' in <[ANNOUNCE] nftables 0.9.1 release>. The problem encountered because Debian Buster using nftables v0.9.0.